lib/region/README.md

```diff
@@ -0,0 +1,38 @@
+# Region
+
+The `region` module is a module meant to standardize the detection of various
+regions, such as EU or APAC. It exposes functions that can test if a locale
```

Comment:
I think APAC is an abbreviation that might need some definition, like `Asia-Pacific (APAC)`.